sql/mob_droplist.sql

```diff
@@ -26780,6 +26780,11 @@ INSERT INTO `mob_droplist` VALUES (3290,0,0,1000,16242,@VCOMMON); -- Dark Ixion:
 INSERT INTO `mob_droplist` VALUES (3290,0,0,1000,2654,@ALWAYS);   -- Dark Ixion: Horn
 INSERT INTO `mob_droplist` VALUES (3290,0,0,1000,2655,@ALWAYS);   -- Dark Ixion: Tail
 
+-- ZoneID: 192 - Magicked Bones
+INSERT INTO `mob_droplist` VALUES (3291,0,0,1000,880,@RARE); --Magicked Bones: Ceremonial Dagger
```

Comment:
Chances are you need to edit spawn points and have 1 of the magicked bones use a different group pool and drop ID than the other. These mobs used to exist in larger quantities in the project, so one of the 2 drop lists _should_ already exists at some other number. (they were probably never separated, or share a list with another skeleton so you might possibly need _two_ for rate purposes..)

The alternate pool would use the opposite model ID.

Also that thing I mentioned about the comments, on some systems the SQL row will fail to import from it.

Reply:
Looks like you were right, the drop pool already existed it just wasn't assigned to the Magicked Bones at all. Well, at least I can remove the extra drop pool unless I need to have the exact same pool twice. As for the models, I was planning on switching to using onMobInitialize like Mowford suggested, assuming they both use the same dropid normally.

Reply:
> As for the models, I was planning on switching to using onMobInitialize like Mowford suggested, assuming they both use the same dropid normally.

pls see my comment warning against that.

_rolls up newspaper_ hey mowford, c'mere a min.